db/seeds.rb

```diff
@@ -1,12 +1,27 @@
 # frozen_string_literal: true
 require "faker"
 
-lambeth = LocalAuthority.find_or_create_by!(name: "Lambeth Council", subdomain: "lambeth")
-southwark = LocalAuthority.find_or_create_by!(name: "Southwark Council", subdomain: "southwark")
+lambeth = LocalAuthority.find_or_create_by!(
+  name: "Lambeth Council",
+  subdomain: "lambeth",
+  signatory_name: "Christina Thompson",
+  signatory_job_title: "Director of Finance & Property",
+  enquiries_paragraph: "Postal address: Planning London Borough of Lambeth PO Box 734 Winchester SO23 5DG.",
+  email_address: "planning@lambeth.gov.uk"
+)
 
-User.find_or_create_by!(email: "admin@example.com") do |user|
+southwark = LocalAuthority.find_or_create_by!(
+  name: "Southwark Council",
+  subdomain: "southwark",
+  signatory_name: "Simon Bevan",
+  signatory_job_title: "Director of Planning",
+  enquiries_paragraph: "Postal address: Planning London Borough of Lambeth PO Box 734 Winchester SO23 5DG.",
```

Comment:
Is this dummy copy, or should we have the actual details for Southwark in here?

Reply:
I copied the details that were in the original email for this one and just pushed them into it. I haven't been given any specific details